src/frontend/Komponenter/Behandling/SettPåVent/SettPåVent.tsx

```diff
@@ -120,6 +119,17 @@ export const SettPåVent: FC<{ behandling: Behandling }> = ({ behandling }) => {
         }).then(settOppgave);
     }, [behandling.id, axiosRequest]);
 
+    const hentOppgavestatusForBehandling = useCallback(() => {
+        axiosRequest<SendtOppgave[], null>({
+            method: 'GET',
+            url: `/familie-ef-sak/api/oppgave/behandling/${behandling.id}/settpavent-oppgavestatus`,
+        }).then((respons: RessursSuksess<SendtOppgave[]> | RessursFeilet) => {
+            if (respons.status === RessursStatus.SUKSESS) {
+                settSendteOppgaver(respons);
+            }
```

Comment:
Trenger ikke if-sjekken her :) Vi vil vel uansett sette responsen, også selv om den feiler
```suggestion
            settSendteOppgaver(respons);
```